.github/workflows/release.yml

```diff
@@ -0,0 +1,17 @@
+name: Releases
+
+on: 
+  push:
+    tags:
+    - '*'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+    - uses: actions/checkout@v3
+    - uses: ncipollo/release-action@v1.14.0
+      with:
+        body: "See: https://github.com/astral-sh/uv/releases/tag/${{ github.ref_name }}"
```

Comment:
Could we just use [`gh release create`](https://cli.github.com/manual/gh_release_create)? I'm generally hesitant to use third-party actions since it's just another dependency to audit.

Reply:
Yea no worries. I will swap it out.

Reply:
It is now changed to use the built in `gh`